FuelSDK-Test/RefreshTokenTest.cs

```diff
@@ -0,0 +1,36 @@
+﻿using NUnit.Framework;
+using System;
+using FuelSDK;
+using Newtonsoft.Json.Linq;
+
+namespace FuelSDK.Test
+{
+    [TestFixture()]
+    public class RefreshTokenTest
+    {
+        ETClient client;
+ 
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            client = new ETClient();
+        }
+
+        [Test()]
+        public void Oauth2RefreshToken()
+        {
+            var token = client.AuthToken;
+            var refreshToken = client.RefreshKey;
+         
+
+            client.RefreshTokenWithOauth2(true);
+            var token1 = client.AuthToken;
+            var refreshToken1 = client.RefreshKey;
+
+
+            Assert.AreNotEqual(token, token1, "Tokens should differ as call is made with Refresh Token");
```

Comment:
If the test name is made more explicit, I don't see the need to add the message parameter in the Assert methods. Plus, the "Tokens should differ as call is made with Refresh Token" should be "Tokens should differ as call to refresh is forced"